tests/console/system_prepare.pm

```diff
@@ -38,6 +38,18 @@ sub run {
         }
     }
 
+    # For media upgrade, need reregistration the modules after migration
+    # poo#54131 [SLE][Migration][SLE12SP5]test fails in system_prepare -
+    # media upgrade need add modules after migration
+    if (get_var('SCC_ADDONS') && is_sle('<15') && get_var('MEDIA_UPGRADE')) {
+        register_product_proxyscc();
+        my $myaddons = get_var('SCC_ADDONS');
+        $myaddons =~ s/ltss,?//g;
```

Comment:
Could you help add comments why do not need ltss

Reply:
Added comments